src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,15 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
```

Comment:
what about the data members? how are you setting them? i don't see a call to `super()` here